apps/sms/services/js/moz_mobile_message/moz_mobile_message_shim.js

```diff
@@ -0,0 +1,210 @@
+/* -*- Mode: js; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- /
+/* vim: set shiftwidth=2 tabstop=2 autoindent cindent expandtab: */
+
+
+/* global bridge, Utils */
+
+/* exported MozMobileMessageShim */
+
+'use strict';
+(function(exports) {
+
+/**
+ * Name of the service for mozMobileMessage API shim
```

Comment:
nit: full stop at the end of jsdoc description. Yeah, I know that I'm becoming too picky on very minor things, just want to have better jsdoc for newly introduced code :)